src/assets/templates/academics_semester_classes.html

```diff
@@ -195,21 +195,36 @@ <h3>Swaps</h3>
             <tbody>
             <tr data-ng-repeat="swapClass in swapClassList">
               <td>
-                <span class="cc-academics-semester-class-swap-number cc-academics-semester-class-swap-number-width">{{swapClass.swapFromCourse.swap_number}}</span><span class="cc-academics-semester-class-swap-number-small-padding">&#x21C4;</span>
+                <span class="cc-academics-semester-class-swap-number cc-academics-semester-class-swap-number-width">{{swapClass.swapFromCourse.swap_number}}</span><span class="cc-academics-semester-class-swap-number-small-padding" aria-hidden="true">&#x21C4;</span>
+                <span class="cc-visuallyhidden" aria-live="polite">{{swapClass.swapFromCourse.swap_number}} Swap</span>
               </td>
               <td>
-              <span>
-                <a data-ng-href="{{course.url}}" data-ng-bind="swapClass.swapFromCourse.course_code">
+                <a data-ng-href="{{swapClass.swapFromCourse.url}}" data-ng-bind="swapClass.swapFromCourse.course_code">
                 </a>
                 <span data-ng-bind="swapClass.swapFromCourse.title"></span>
               </td>
               <td>
-                <a data-ng-href="{{course.url}}" data-ng-bind="swapClass.swapToCourse.course_code">
-                </a>
-                <span data-ng-bind="swapClass.swapToCourse.title"></span>
+                <span>
+                  <a data-ng-href="{{swapClass.swapToCourse.url}}" data-ng-bind="swapClass.swapToCourse.course_code">
+                  </a>
+                  <span data-ng-bind="swapClass.swapToCourse.title"></span>
+                </span>
+                <span data-ng-repeat="swapAdditional in swapClass.swapToAdditionalCourse">
+                  <br>
+                  <span>OR</span>
+                  <br>
+                  <a data-ng-href="{{swapAdditional.url}}" data-ng-bind="swapAdditional.course_code">
+                  </a>
+                  <span data-ng-bind="swapAdditional.title"></span>
+                </span>
               </td>
               <td>
-                <span data-ng-bind="swapClass.dateRequested"></span>
+                <span data-ng-bind="swapClass.swapToSection.last_enrl_dt_stmp"></span>
+                <span data-ng-repeat="swapAdditionalSec in swapClass.swapToAdditionalSection">
+                  <br>
+                  <br>
```

Comment:
@redconfetti, @developish , @nkutub ,
Is it allowed/"normal" for us to use
```
<br>
<br>
```
in this manner?

To be clear I'm not _objecting_, I just want to be sure what he is doing is _right_?

Reply:
@marcpechaitis  Excellent observation. Actually, I thought it was ok to remove them. Then I found that it was used for handling the second column which may have multiple swap classes with a OR line. So the extra <br>'s ensure that the lines are aligned correctly.  Please refer to the specification attached to the ticket (SISRP-46442 v4.pdf). You will see the "Date Requested" are aligned to handle the multiple courses with OR:

CHEM ENG 24                      August 15, 2019 <br>
OR                                         <br>
CHEM ENG 26                      August 17, 2019